crates/cli/src/main.rs

```diff
@@ -52,6 +51,29 @@ fn main() -> Result<()> {
 
             let wasm = gen.generate(&js)?;
 
+            fs::write(&opts.output, wasm)?;
+            Ok(())
+        }
+        Command::Build(opts) => {
+            let js = JS::from_file(&opts.input)?;
+            let codegen: CodegenOptionGroup = opts.codegen.clone().into();
+            let mut builder = CodeGenBuilder::new();
+            builder
+                .wit_opts(WitOptions::from_tuple((
+                    codegen.wit.clone(),
+                    codegen.wit_world.clone(),
+                ))?)
+                .source_compression(!codegen.no_source_compression)
+                .provider_version("2");
```

Comment:
I figured we'd bump this in a future PR (probably the one with where I add the `-J` flag) since we're not making any changes to the provider in this PR and there's already a lot going on.